pages/Modes.md

```diff
@@ -85,7 +85,7 @@ supported in that mode. A number indicates the footnotes describe additional
 information.
 
     | Option                 | type    | :null   | :strict | :compat | :rails  | :object | :custom |  :wab   |
-    | ---------------------- | ------- | ------- | ------- | ------- | ------- | ------- | ------- | ------- |
+    | -----------------------| ------- | ------- | ------- | ------- | ------- | ------- | ------- | ------- |
```

Comment:
Just noticed this. A space is needed between the - and |.

Reply:
Fixed